app/src/main/res/drawable/camera_24.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="25">
+  <path
+      android:pathData="M5.469,6.5C4.14,6.5 3.063,7.545 3.063,8.833V16.167C3.063,17.455 4.14,18.5 5.469,18.5H13.031C14.36,18.5 15.438,17.455 15.438,16.167V14.906C15.438,14.548 15.634,14.218 15.953,14.04C16.272,13.861 16.665,13.861 16.984,14.04L20.938,16.253V8.747L16.984,10.96C16.665,11.139 16.272,11.139 15.953,10.96C15.634,10.782 15.438,10.452 15.438,10.094V8.833C15.438,7.545 14.36,6.5 13.031,6.5H5.469ZM21.195,16.397L21.195,16.397L21.195,16.397ZM1,8.833C1,6.44 3.001,4.5 5.469,4.5H13.031C15.34,4.5 17.24,6.198 17.475,8.376L20.164,6.871C21.424,6.165 23,7.047 23,8.459V16.542C23,17.953 21.424,18.835 20.164,18.129L17.475,16.624C17.24,18.802 15.34,20.5 13.031,20.5H5.469C3.001,20.5 1,18.56 1,16.167V8.833Z"
+      android:fillColor="?attr/primaryIconColor"
+      android:fillAlpha="0.84"
```

Comment:
this has to be removed, the alpha is already in the color. Are you extracting the assets from Figma? You should get them from [Dub](https://dub.duckduckgo.com/duckduckgo/Icons/tree/master/Glyphs)